src/helpers/database/schema.sql

```diff
@@ -145,3 +145,15 @@ CREATE TABLE subscriptions (
   INDEX ipfs (ipfs),
   INDEX created (created)
 );
+
+
+CREATE TABLE users (
+  id VARCHAR(66) NOT NULL,
```

Comment:
What's the id? Isn't the same as address? In this case I would not add address here but just id, and use it as primary key